README.md

```diff
@@ -102,12 +102,21 @@ Such a build requires the presence of [CUnit](http://cunit.sourceforge.net/).  Y
 yourself, or you can choose to instead rely on the [Conan](https://conan.io) packaging system that
 the CI build infrastructure also uses.  In that case, install Conan and do:
 
-    $ conan install ..
+    $ conan install .. --build missing
+
+in the build directory prior to running cmake.
+
+The CUnit Conan package is hosted in the [Bintray repository](https://bintray.com/bincrafters/public-conan).
```

Comment:
Perhaps better to specify *Bincrafters Bintray repository*? Like with git you can have multiple remotes and we explicitly depend on the Bincrafters for CUnit.